crates/ruff_notebook/src/notebook.rs

```diff
@@ -408,13 +408,18 @@ impl Notebook {
         &self.raw.metadata
     }
 
-    /// Return `true` if the notebook is a Python notebook, `false` otherwise.
+    /// Check if it's a Python notebook.
+    ///
+    /// This is determined by checking the `language_info` or `kernelspec` in the notebook
+    /// metadata. If neither is present, it's assumed to be a Python notebook.
     pub fn is_python_notebook(&self) -> bool {
-        self.raw
-            .metadata
-            .language_info
-            .as_ref()
-            .map_or(true, |language| language.name == "python")
+        if let Some(language_info) = self.raw.metadata.language_info.as_ref() {
+            return language_info.name == "python";
+        }
+        if let Some(kernel_spec) = self.raw.metadata.kernelspec.as_ref() {
+            return kernel_spec.language.as_deref() == Some("python");
+        }
+        true
```

Comment:
I think defaulting to `true` is good for most notebooks. If this proves a problem, we can look at specific notebooks or decide to introduce an option.

Reply:
Sounds good.